src/main/java/it/gov/pagopa/payment/connector/rest/reward/mapper/RewardCalculatorMapper.java

```diff
@@ -32,6 +33,11 @@ public AuthPaymentRequestDTO rewardMap(TransactionInProgress transactionInProgre
                 .channel(transactionInProgress.getChannel())
                 .build();
     }
+    public AuthPaymentRequestDTO authRequestMap(TransactionInProgress transactionInProgress) {
+        return AuthPaymentRequestDTO.builder()
```

Comment:
this body is totally empty except the rewardCents field!
you could write a common method to call in order to fill the model in both methods...